cpp/data/installer/ub_tools.sql

```diff
@@ -39,12 +39,14 @@ CREATE TABLE zeder_journals (
 
 -- Table to be used w/ our validate_harvested_records tool:
 CREATE TABLE metadata_presence_tracer (
-    zeder_journal_id INT(11) UNSIGNED,
+    journal_id INT(11) UNSIGNED,
     marc_field_tag CHAR(3) NOT NULL,
+    marc_subfield_code CHAR(1) NOT NULL,
+    optional_regex VARCHAR(200),
```

Comment:
- I don't remember any other field that contains the prefix "optional_" in its name, just because it is not mandatory. Is this really necessary?
- How about a DEFAULT value for an optional field?

Reply:
Renamed

Reply:
optional_regex was renamed to regex in the schema, but i can still find at least 5 occurrences in the code (e.g. when building sql statements) where it's still named optional_regex.

Reply:
Fixed